x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -111,6 +113,18 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
     }
   };
 
+  // Regression supports numeric fields. Classification supports numeric, boolean, text, keyword and ip.
+  const shouldAddFieldOption = (field: Field) => {
```

Comment:
Boolean types aren't currently returned from the `newJobCapsService`. You will need to add `ES_FIELD_TYPES.BOOLEAN` into the list of supported types in `server/models/job_service/new_job_caps/field_service.ts`, and in turn some of the tests will need editing.

Reply:
Updated `newJobCapsService` in c09aa277087cc5f4466d8d2bd1ba19898b890800